oak_restricted_kernel_sdk/src/dice.rs

```diff
@@ -84,3 +94,13 @@ fn get_restricted_kernel_dice_data() -> anyhow::Result<RestrictedKernelDiceData>
     }
     Ok(result)
 }
+
+pub struct Signer {
```

Comment:
nit: I also think that `Signer` should just be in the `oak_restricted_kernel_sdk`, not the `oak_restricted_kernel_sdk::dice`